velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -1704,13 +1704,6 @@ DEBUG_ONLY_TEST_F(
             pool->reclaimer()->leaveArbitration();
           })));
 
-  // Verifies that we only trigger the hash build reclaim once.
-  std::atomic<int> numHashBuildReclaims{0};
-  SCOPED_TESTVALUE_SET(
-      "facebook::velox::exec::HashBuild::reclaim",
```

Comment:
Should we also remove this from HashBuild::reclaim? Are there other tests that still use it? Are they flaky as well?

```
  TestValue::adjust("facebook::velox::exec::HashBuild::reclaim", this);
```